### ISSUE TITLE
Experiment table, graphs, are several hours behind output.

### ISSUE THREAD

**Issue by bw4sz** [comet-ml/issue-tracking#75]
I have a fairly long running experiment (16 hrs) [here](https://www.comet.ml/bw4sz/deepforest-retinanet). The graphs are quite behind the actual experiment.

For example, the graphs and metrics are still from epoch 2
![image](https://user-images.githubusercontent.com/1208492/42741353-4d19230c-8866-11e8-829d-3113c97977b1.png)

when the actual run is at epoch 4

![image](https://user-images.githubusercontent.com/1208492/42741363-69ea0b22-8866-11e8-8e97-1564d96243cd.png)

each epoch takes about 2 hours to run.

I don't think i'm limited by upload. I'm on a University HPC system, but let me know if there is something on my end. Let me know if my expectations are out of sync with project goals.

**Comment by bw4sz** [comment]
public link. 

https://www.comet.ml/bw4sz/deepforest-retinanet/6463e304ba4642b7874009d579c6d415

It could have something to do with image upload. See #70 , I think because its trying to load 4 * 20 images?

**Comment by rememberlenny** [comment]
Thanks @bw4sz - We are taking a look.

**Comment by bw4sz** [comment]
I haven't test rigorously, but current runs look much better.

**Comment by Nimrod007** [comment]
@bw4sz - im aware of this issue. rolling a fix into production. since we are in high volume of experiment running in parallel this will take some time to resolve. ill keep you updated

**Comment by bw4sz** [comment]
No rush, just updating here. Before, it appeared that while the graphs were behind, they would eventually catch up after the run. This wasn't rigorously tested, but appeared to be true.

I had a run yesterday that was behind, then finished, but never updated.

https://www.comet.ml/bw4sz/deepforest-retinanet/93041bc04c1c4a7aac5e13cf7d134caa

both graphs, hyperparameter and metrics tabs are all behind output. Its clear when you look at the mAP metric

**Comment by Nimrod007** [comment]
thanks for updating @bw4sz - were still on that issue.

**Comment by ross-development** [comment]
I'm seeing a similar issue where the results of my experiments (e.g. the values in the Metrics and Hyper Parameters tab on the Comet ML UI) only show up anywhere from 30 secs to 5 mins after the experiment successfully finished. But, in a significant number of cases, the missing values never show up at all, which is really problematic. The experiment itself does show up, but many of the values associated to it are missing.

Was gonna create a separate issue for this, but it seems it is related to this thread as it is.

**Comment by Nimrod007** [comment]
fixed.
